run_tests.py

```diff
@@ -368,12 +368,8 @@ def __check_kmemleak() -> None:
 remote.connect()
 
 if run_config.KERNEL_DBG_TESTS:
-    try:
-        remote.tempesta.run_cmd("cat /sys/kernel/debug/kmemleak")
-    except remote.CmdError:
-        raise AttributeError(
-            "kmemleak is disabled. Please enable kmemleak or not use --kernel-dbg option."
-        )
+    remote.tempesta.run_cmd("cat /sys/kernel/debug/kmemleak")
```

Comment:
I don't think that's a good idea. Why delete a recommendation message? Not everyone knows how kmemleak works and will immediately understand the reason for this exception

Reply:
Catching and re-reaising an exception without any actions is not good practice (it was implemented in this way.).
Plus, error may be different and it may not to be related to `kmemleak is disabled`. **We do not need to hide an initial message.**  

I agree that having an explanation is a good idea but the assumption is based on almost nothing. For such things I would create something for complex to determine the reason of the command fail.

BTW, if you want, I can easily update it.

Reply:
But the script will return a non-0 status code (`ProcessBadExitStatusException` exception) if the file does not exist, otherwise we will receive `ProcessKilledException` or `CommandExecutionException`.

Reply:
Updated with a new exception.